packages/components/src/components/button/_button.scss

```diff
@@ -79,19 +80,19 @@
       background: transparent;
       color: $disabled;
 
-      & > .#{$prefix}--btn__icon {
+      & > .#{$prefix}--btn__icon path {
         fill: $disabled;
       }
     }
 
-    &:hover > .#{$prefix}--btn__icon {
+    &:hover > .#{$prefix}--btn__icon path {
```

Comment:
what about icons that use other elements other than `path` such as `circle`, `rect`, etc.

Reply:
I believe that is based on the existing styles which also target `path` but to address your concerns I am selecting `svg` now